File: easy-solution/minesp3164/485.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        max_count = 0
        count = 0
        for i in range(len(nums)):
            if(nums[i] == 1):
                count = count + 1
            else:
                max_count = max(max_count, count)
                count = 0
        return max_count

File: easy-solution/Antidote21/485.c
int findMaxConsecutiveOnes(int* nums, int numsSize) {
	int i, max = 0, cnt = 0;
	for (i = 0; i < numsSize; i++) {
		if (nums[i] == 1) {
			cnt++;
			if (max < cnt) max = cnt;
		}
		else cnt = 0;
	}
	return max; 
}
